src/plugins/blocksmith/components/blocksmith/index.jsx

```diff
@@ -844,67 +827,19 @@ class Blocksmith extends React.Component {
   /**
    * Handles the custom command type(s) sent by keyBindingFn.
    *
-   * @link https://draftjs.org/docs/advanced-topics-key-bindings.html
+   * @link https://draftjs.org/docs/advanced-topics-key-bindings
    *
    * @param {string} command - a command type
    *
    * @returns {string} 'handled' (meaning we did something special) or 'not-handled' (meaning we
    * want to allow the editor to take over and resume default behavior).
    */
   handleKeyCommand(command) {
-    const { activeBlockKey, editorState } = this.state;
-    let anchorKey;
-    let currentBlock;
-    let newBlockKey;
+    const { editorState } = this.state;
     let newContentState;
```

Comment:
@efjacobson - quick suggestion here: if you'd like to declare variables that are used in a specific switch case clause and not be available to the entire switch block, then you can wrap that clause in curly brackets. Currently, the three variables are needed only for the first case, so it keeps clean to have them just inside that one case.

Reply:
sweet, did not know that. updated